src/main/java/frc/robot/commands/AutonDriveCommand.java

```diff
@@ -56,7 +56,7 @@ public void execute() {
       met = false;
     }
     System.out.format("Target Distance:%.02f\n", distance);
-    drive_subsystem.drive(moveRequest, turnRequest);
+    drive_subsystem.drive(moveRequest * 0, turnRequest);
```

Comment:
Needs a comment why this is times 0.  Is it a placeholder?  Could it just be 0?  Why?